### DISCUSSION TITLE
Information on Snapshot

### DISCUSSION THREAD

**Discussion by d2e** [microsoft/FluidFramework#20419] [Q&A] [answered]
Could some one help in sharing insight on how snapshot works? what is the difference between summary and snapshot?
And when snapshot is used by the fluid client

**Comment by d2e** [comment]
@nmsimons

**Comment by kashms** [comment, marked as answer]
You can find more details about summaries and snapshots in the FAQ section: https://fluidframework.com/docs/faq/#summarization
